modules/cloudsql/main.tf

```diff
@@ -47,9 +47,10 @@ resource "random_password" "pwd" {
 }
 
 resource "google_sql_user" "cloudsql_user" {
-  name     = var.db_user
-  instance = google_sql_database_instance.main.name
-  password = random_password.pwd.result
+  name            = var.db_user
+  instance        = google_sql_database_instance.main.name
+  password        = random_password.pwd.result
+  deletion_policy = "ABANDON"
```

Comment:
TY for fixing! fyi @blackzlq @artemvmin this fixes the issue with cloud SQL instance deletion.
@umeshkumhar Can you leave a comment explaining why this was added so it doesn't get accidentally deleted in the future?